github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1885,3 +1885,43 @@ periodics:
         privileged: true
     nodeSelector:
       type: bare-metal-external
+- name: periodic-kubevirt-image-bump
+  cron: "5 7 * * 6"
+  max_concurrency: 1
+  annotations:
+    testgrid-create-test-group: "false"
+  labels:
+    preset-gcs-credentials: "true"
+    preset-github-credentials: "true"
+    preset-bazel-unnested: "true"
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: main
+    workdir: true
+  - base_ref: main
+    org: kubevirt
+    repo: project-infra
+  cluster: ibm-prow-jobs
+  spec:
+    securityContext:
+      runAsUser: 0
+    containers:
+    - image: quay.io/kubevirt/builder:2212180911-8818abcfa
```

Comment:
So the bootstrap image should already contain `skopeo` and `bazelisk`. Also you have already `preset-bazel-unnested`.

So from that perspective, can't you yous:

 1. the bootstrap image here
 2. simplify the args to:
 
 ```
 - |
   make bump-images
   ../project-infra/hack/git-pr.sh -c "cd ../kubevirt && bazelisk run //robots/cmd/uploader:uploader -- -workspace ./WORKSPACE -dry-run=false" -p ../kubevirt -r kubevirt -L lgtm,approved,release-note-none -T main
 ```

Reply:
Makes sense :+1:

Reply:
@rmohr bootstrap doesn't contain the pr-creator binary. I could use the pr-creator image which is based on the golang image, will try that.